src/app/pages/terms-of-service/terms-of-service.component.html

```diff
@@ -1,11 +1,18 @@
 <div class='container'>
   <h1 class='text-center mt-3 mb-3'>Terms of Service</h1>
   <div class='alert alert-primary' role='alert'>
-    Last updated: June 27, 2020
+    Last updated: July 15, 2020
   </div>
+  <h3>SUMMARY</h3>
+  <p>In short, you agree to abide by our code of conduct for safe, respectful, and common-sense behaviour. You
+    understand that we will do everything in our power to address reported violations of the code or these Terms,
+    including account termination at our discretion, but that the ultimate responsibility for a message's contents,
+    including links to third-party sites, lies with the user who sent it. You agree not to use unlicenced or alter
+    any of our intellectual property, and where necessary, to obtain parental consent to use our site.
```

Comment:
> You agree not to use unlicensed... **what?**

Also, should you be more specific and say that you can't use unlicensed material or material which you do not have the right to distribute?

Reply:
```suggestion
    including links to third-party sites, lies with the user who sent it. You agree not to use reproduce or alter
    any of our intellectual property for unlicensed use, and where necessary, you agree to obtain parental
    consent to use our site.
```

Reply:
This refers the IP related to the site itself. *Posting* unlicensed material is subsumed under adherence to the code of conduct and user responsibility for message content, including images. It *is* a brief high-level summary

Reply:
@anthonyftwang add commas between `use reproduce or alter` and commit that, sounds good to me.